### DISCUSSION TITLE
Should we add indices as the alias of indexes in `show indexes`?

### DISCUSSION THREAD

**Discussion by wey-gu** [vesoft-inc/nebula#3402] [General]
- They are both valid:
  - https://www.nasdaq.com/articles/indexes-or-indices-whats-the-deal-2016-05-12
- The plural of the vertex is of two shapes, too, and we had chosen `vertices`
  - https://www.wordhippo.com/what-is/the-plural-of/vertex.html

**Comment by Shylock-Hg** [comment]
Maybe we could  reference practice in other DBs.

**Reply by wey-gu** [reply]
> Maybe we could reference practice in other DBs.

Thanks @Shylock-Hg , all DBs are using indexes :), will close the discusison
